### DISCUSSION TITLE
ffmpeg support Please!

### DISCUSSION THREAD

**Discussion by skynight137** [anasty17/mirror-leech-telegram-bot#936] [Q&A]
I'm trying install from docker or command /shell always show
'''
Stderr
/bin/sh: 1: ffmpeg: not found
'''

and this is output from update -y
Stdout
Get:1 http://security.ubuntu.com/ubuntu jammy-security InRelease [110 kB]
Get:2 http://archive.ubuntu.com/ubuntu jammy InRelease [270 kB]
Get:3 http://archive.ubuntu.com/ubuntu jammy-updates InRelease [114 kB]
Get:4 http://archive.ubuntu.com/ubuntu jammy-backports InRelease [99.8 kB]
Get:5 http://security.ubuntu.com/ubuntu jammy-security/universe amd64 Packages [132 kB]
Get:6 http://archive.ubuntu.com/ubuntu jammy/multiverse amd64 Packages [266 kB]
Get:7 http://security.ubuntu.com/ubuntu jammy-security/restricted amd64 Packages [313 kB]
Get:8 http://security.ubuntu.com/ubuntu jammy-security/main amd64 Packages [333 kB]
Get:9 http://archive.ubuntu.com/ubuntu jammy/restricted amd64 Packages [164 kB]
Get:10 http://archive.ubuntu.com/ubuntu jammy/universe amd64 Packages [17.5 MB]
Get:11 http://security.ubuntu.com/ubuntu jammy-security/multiverse amd64 Packages [4,644 B]
Get:12 http://archive.ubuntu.com/ubuntu jammy/main amd64 Packages [1,792 kB]
Get:13 http://archive.ubuntu.com/ubuntu jammy-updates/restricted amd64 Packages [354 kB]
Get:14 http://archive.ubuntu.com/ubuntu jammy-updates/multiverse amd64 Packages [7,791 B]
Get:15 http://archive.ubuntu.com/ubuntu jammy-updates/main amd64 Packages [641 kB]
Get:16 http://archive.ubuntu.com/ubuntu jammy-updates/universe amd64 Packages [264 kB]
Get:17 http://archive.ubuntu.com/ubuntu jammy-backports/universe amd64 Packages [5,814 B]
Fetched 22.3 MB in 3s (7,637 kB/s)
Reading package lists...
Reading package lists...
Building dependency tree...
Reading state information...
ffmpeg is already the newest version (7:4.4.2-0ubuntu0.22.04.1).
0 upgraded, 0 newly installed, 0 to remove and 3 not upgraded.

**Comment by anasty17** [comment]
What do u want ? 
This this the repo use it as it is. 
 U can't ask for edits. Try to know by ur self

**Reply by skynight137** [reply]
ffmpeg for video editing?
yeah, instead of doing it repeatedly(dl/up) , it will take up a lot of resources.  e.g. I want to cut/merge/watermaking... I hope it supports ffmpeg even if it's only accessible in /shell no problem.

**Reply by anasty17** [reply]
Huh u can't cut/merge while up or down! All people wants all in one bot ! Use another bot for this purpose! What about adding weather status in state ?

**Reply by skynight137** [reply]
plan: put code in `class TgUploader`.  precisely when uploading files (video/audio), here the ffmpeg code is executed.

**Reply by anasty17** [reply]
Video editing is another project
